### PR TITLE
Make shocker zombies emit noises

### PR DESCRIPTION
**Pull request by AnotherSeawhite**
#### Summary
Balance "Add sounds for shockers"

#### Purpose of change
To let the people know that there's a threat nearby. It's not really great when you get zapped by a shocker that came out of nowhere.

#### Describe the solution
Add `PARROT` special attacks for shockers

#### Describe alternatives you've considered
None

#### Testing
Made a new world, Created a character, spawned in, debug spawned the shockers, waited for a few turns and the shockers emit noises

#### Additional context
Don't you hate it when you open a door just to get pranked by a shocker brute?

### CODE REVIEW

File: data/json/speech.json [hunk 1]
```diff
@@ -2735,5 +2735,17 @@
...
+  {
+    "type": "speech",
+    "speaker": [ "mon_zombie_brute_shocker", "mon_zombie_electric", "mon_zombie_nullfield", "mon_skeleton_electric" ],
+    "sound": "zapping noises.",
+    "volume": 20
+  },
+  {
+    "type": "speech",
+    "speaker": [ "mon_zombie_brute_shocker", "mon_zombie_electric", "mon_zombie_nullfield", "mon_skeleton_electric" ],
+    "sound": "electrical sparks.",
```
Comment (MNG-cataclysm) [inline data/json/speech.json:2748]:
```suggestion
    "sound": "high-pitch hum.",
```
You can't really "hear" electrical sparks.

File: data/json/speech.json [hunk 2]
```diff
@@ -2735,5 +2735,17 @@
     "speaker": [ "mon_feral_granny" ],
     "sound": "\"That's my purse!  I don't know you!\"",
     "volume": 20
+  },
+  {
+    "type": "speech",
+    "speaker": [ "mon_zombie_brute_shocker", "mon_zombie_electric", "mon_zombie_nullfield", "mon_skeleton_electric" ],
+    "sound": "zapping noises.",
```
Comment (MNG-cataclysm) [inline data/json/speech.json:2742]:
```suggestion
    "sound": "crackling sounds.",
```

**Review by MNG-cataclysm — CHANGES_REQUESTED** [review]

File: data/json/speech.json (hunk 1 above)
Comment (Zireael07) [reply data/json/speech.json:2748]: High-pitchED, I think.

Comment (NotSeawhite) [reply data/json/speech.json:2748]: I have no knowledge about those, so.. yeah :P

**Pull request merged**